plutus-scb/src/Plutus/SCB/Core.hs

```diff
@@ -73,41 +76,54 @@ import           Wallet.Effects                   (ChainIndexEffect, NodeClientE
 
 type ContractEffects t =
         '[ EventLogEffect (ChainEvent t)
-         , Log
+         , LogMsg Text
          , UUIDEffect
          , ContractEffect FilePath
          , Error SCBError
          ]
 
+data CoreMsg =
+    Installing (Doc Void)
```

Comment:
:thinking: 
Is this just to avoid the `t` parameter?

Reply:
Yes that's what it was for, but I've come around to your point of view (that we will need the `t` sooner or later anyway)